konfera/tests/test_models.py

```diff
@@ -409,3 +409,17 @@ def test_status(self):
         tt.date_from = now - datetime.timedelta(days=3)
         tt.date_to = now - datetime.timedelta(days=1)
         self.assertEquals(tt.status, TicketType.STATUSES[TicketType.EXPIRED])
+
+
+class OrganizerTest(TestCase):
+
+    def setUp(self):
+        self.first_organizer = models.Organizer(title='Mysterious Organizer', street='1 Up street', city='Big City',
+                                                company_id='123',
+                                                about_us="World famous yet unknown conference organizer.")
+        self.noname_organizer = models.Organizer(street='2 Random', city='Small City')
```

Comment:
You created the `noname_organizer`` to test it cannot be saved. So I think the only place we will use it is in `test_save` so please move it to that method as a local variable.